packages/@aws-cdk/aws-stepfunctions/lib/condition.ts

```diff
@@ -2,13 +2,110 @@
  * A Condition for use in a Choice state branch
  */
 export abstract class Condition {
+
+  /**
+   * Tests if variable is present
+   */
+  public static isPresent(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsPresent, true);
+  }
+
+  /**
+   * Tests if variable is not present
+   */
+  public static isNotPresent(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsPresent, false);
+  }
+
+  /**
+   * Tests if variable is a string
+   */
+  public static isString(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsString, true);
+  }
+
+  /**
+   * Tests if variable is not a string
+   */
+  public static isNotString(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsString, false);
+  }
+
+  /**
+   * Tests if variable is numeric
+   */
+  public static isNumeric(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsNumeric, true);
+  }
+
+  /**
+   * Tests if variable is not numeric
+   */
+  public static isNotNumeric(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsNumeric, false);
+  }
+
+  /**
+   * Tests if variable is boolean
+   */
+  public static isBoolean(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsBoolean, true);
+  }
+
+  /**
+   * Tests if variable is not boolean
+   */
+  public static isNotBoolean(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsBoolean, false);
+  }
+
+  /**
+   * Tests if variable is a timestamp
+   */
+  public static isTimestamp(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsTimestamp, true);
+  }
+
+  /**
+   * Tests if variable is not a timestamp
+   */
```

Comment:
the terminology `tests` and `matches` are used. can we be consistent (I think I prefer Matches which already exists on a the comparison operators that are available today)

Reply:
happy with that